clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
```

Comment:
i am not married to this name. `cljr--in-keyword-sans-alias?` it is then.

apparently it is consistent with the cider naming (already ;) ):
```
(not (cider-in-comment-p))
(not (cider-in-string-p))
```

not married to this either tho. happy to rename if you insist.